src/latexify/frontend.py

```diff
@@ -80,74 +95,67 @@ def get_latex(
         ).visit(tree)
 
 
-class LatexifiedFunction:
-    """Function with latex representation."""
+@overload
+def algorithmic(alg: Callable[..., Any], **kwargs: Any) -> output.LatexifiedAlgorithm:
```

Comment:
Better to use the same argument name.
```suggestion
def algorithmic(fn: Callable[..., Any], **kwargs: Any) -> output.LatexifiedAlgorithm:
```